src/Elmah.Io.ElasticSearch/Elmah.Io.ElasticSearch.nuspec

```diff
@@ -2,21 +2,23 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <version>$version$</version>
-    <authors>elmah.io,jhilden</authors>
+    <authors>elmah.io,jhilden,Asgeir</authors>
     <owners />
     <dependencies>
       <dependency id="elmah" version="1.2.2" />
       <dependency id="elmah.corelibrary" version="1.2.2" />
-      <dependency id="NEST" version="2.4.5" />
+      <dependency id="NEST" version="6.2.0" />
+      <dependency id="Elasticsearch.Net" version="6.2.0" />
+      <dependency id="Newtonsoft.Json" version="11.0.2" />
     </dependencies>
-    <id>Elmah.ElasticSearch</id>
-    <title>Elmah.ElasticSearch</title>
+    <id>HRMTS.Elmah.ElasticSearch</id>
```

Comment:
project url to https://github.com/hrmts/Elmah.Io.ElasticSearch can also be removed...